src/components/ContentListTable/components/AddContent/AddContent.tsx

```diff
@@ -389,7 +389,7 @@ const AddContent = ({ isLoading }: Props) => {
             </Stack>
           }
         >
-          <TableComposable ouiaId='modal_table_expandable' aria-label='Expandable table'>
+          <TableComposable ouiaId='modal_table_expandable_to_add_repo' aria-label='Expandable table for repo add modal'>
```

Comment:
Is this all on one line style better?

Reply:
moved before I could comment - before it was outside of the element, so it was invalid syntax.